src/utils/url.js

```diff
@@ -241,3 +241,12 @@ export function prepareUrl (url) {
     // For example, the "//////google.com" url is equal to "//google.com".
     return url.replace(/^\/+(\/\/.*$)/, '$1');
 }
+
+export function fixUrlAfterResolving (urlBefore, urlAfter) {
```

Comment:
It should be `ensureTrailingSlash` or something like that. I also don't like arg names. Maybe `srcUrl` and `processedUrl` or smthg like this. 
\cc @VasilyStrelyaev 
Overall this feels extremely hacky, mostly because currently URL-related stuff is a completely fucked up. But  for now, this is just fine